contrib/kserve/tests/requirements.txt

```diff
@@ -0,0 +1,4 @@
+pytest>=7.0.0
+kserve>=0.8.0
```

Comment:
Not sure if change is necessary here, but noticed that this version is different from Kserve version from the makefile

Reply:
Yes, we upgraded the manifests while we were working on the PR. This has to change as well.